client.go

```diff
@@ -390,6 +392,31 @@ type Client struct {
 // Initialize a default *Client instance
 var DefaultClient = newClient(nil)
 
+func (c *Client) SetIgnoreErrors(errs []string) error {
+	joinedRegexp := strings.Join(errs, "|")
+	r, err := regexp.Compile(joinedRegexp)
+	if err != nil {
+		return fmt.Errorf("failed to compile regexp %q for %q: %v", joinedRegexp, errs, err)
+	}
+
+	c.mu.Lock()
+	c.ignoreErrorsRegexp = r
+	c.mu.Unlock()
+	return nil
+}
+
+func (c *Client) shouldExcludeErr(errStr string) bool {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	if c.ignoreErrorsRegexp != nil && c.ignoreErrorsRegexp.MatchString(errStr) {
```

Comment:
Just a nitpick, then I'll merge:

This can just be:

```go
return c.ignoreErrorsRegexp != nil && c.ignoreErrrosRegexp.MatchString(errStr)
```

This already is a Boolean, so you can just return it directly. As is, this is like doing:

```go
a := true
if a {
  return true
}
return false
```